arduino/include/arduino.h

```diff
@@ -0,0 +1,179 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include "serialib.h"
+#include <ios>
+#include <thread>
+class Arduino
+{
+public:
+   enum HEADERS
+   {
+      GET_FLOAT = 700,
+      GET_BOOLEAN = 701,
+      ASK_FOR_STATUS_PER_SENSOR = 30 /* the sensor*/,
```

Comment:
what is the meaning of this comment? comments should explain unclear code. There shouldn't be unclear comment for no reason, if there is please add a real comment.

Reply:
fixed